web/cobrands/zurich/_bikeable_banner.scss

```diff
@@ -0,0 +1,36 @@
+#bikeable-banner {
+    background-color: #99C32E;
+    color: white;
+    text-align: center;
+
+    p {
+        margin: 0;
+        padding: 0.5em;
+    }
+
+    @media screen and (min-width: 48em) {
+        position: absolute;
+        margin: 0;
+        transform: rotate(-15deg);
+
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        padding: 1em;
+        width: 150px;
+        height: 150px;
+        border-radius: 150px;
+        top: 120px;
+        left: 6px;
+    }
+
+    @media screen and (min-width: 1025px) {
```

Comment:
If being really pinickity, this changes at a point where the circle then goes off the edge of the screen for a little bit, which would be fixed if this min-width was a few pixels more, I think. But don't think it matters